<?xml version="1.0"?>
<launch>
  <arg name="model" default="custom"/>
  <arg name="yolo_directory" default="$(find villa_yolo)/data/models/$(arg model)"/>
  
  <arg name="labels_path" default="$(arg yolo_directory)/model.names"/>
  <arg name="cfg_path" default="$(arg yolo_directory)/model-evaluate.cfg"/>
  <arg name="weights_path" default="$(arg yolo_directory)/model.weights"/>

  <!-- Used compressed image stream -->
  <param name="image_transport" value="compressed" />

  <node pkg="villa_object_cloud" type="yolocloud_node" name="yolocloud_node" output="screen">
    <param name="yolo_labels" value="$(arg labels_path)" />
    <param name="yolo_cfg" value="$(arg cfg_path)" />
    <param name="yolo_weights" value="$(arg weights_path)" />
  </node>
</launch>
